.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -57,8 +57,9 @@ body:
     id: homeserver
     attributes:
       label: Homeserver
-      description: Which server is your account registered on?
-      placeholder: e.g. matrix.org
+      description: |
+        Which server is your account registered on? If it is a local or non public homeserver, please tell us what is the homeserver implementation (ex: Synapse/Dendrite/etc.) and its version.
```

Comment:
local or non-public homeserver… and the version

Reply:
done